web/packages/teleport/src/Assist/Sidebar/Sidebar.tsx

```diff
@@ -46,34 +50,73 @@ const ChatHistoryTitle = styled.div`
   margin-bottom: 13px;
 `;
 
+const ChatHistoryItemButtons = styled.div`
+  position: absolute;
+  right: 5px;
+  top: 0;
+  bottom: 0;
+  display: none;
+  align-items: center;
+  justify-content: center;
+`;
+
+const ChatHistoryItemDeleteButton = styled.div`
+  cursor: pointer;
+  padding: 5px 5px 0;
```

Comment:
FWIW, it's best to use values for spacing that come directly from the theme to maintain a cohesive look throughout the app. Look for the uses of [`theme.space`](https://github.com/gravitational/teleport/blob/2083dc810985c0f9928c65fe10f64a207cccedb0/web/packages/design/src/theme/darkTheme.ts#L23) throughout the app.

Something to adjust in the future ofc, not in this PR.